spec/integration/base_spec.rb

```diff
@@ -1,79 +1,32 @@
 require 'spec_helper'
 
 describe ActiveFedora::Base do
-  describe "A base object with metadata" do
-    before :each do
-      class MockAFBaseRelationship < ActiveFedora::Base
-        has_metadata 'foo', type: Hydra::ModsArticleDatastream
+  describe '#reload' do
+    before do
+      class Foo < ActiveFedora::Base
+        property :person, predicate: ::RDF::Vocab::DC.creator
       end
     end
-    after :each do
-      Object.send(:remove_const, :MockAFBaseRelationship)
+    after do
+      Object.send(:remove_const, :Foo)
     end
-    describe "a new document" do
+    context "when persisted" do
+      let(:object) { Foo.create(person: ['bob']) }
+      let(:object2) { Foo.find(object.id) }
       before do
-        @obj = MockAFBaseRelationship.new
-
-        @obj.foo.person = "bob"
-        @obj.save
-      end
-
-      it "saves the datastream." do
-        obj = described_class.find(@obj.id)
-        expect(obj.foo).to_not be_new_record
-        expect(obj.foo.person).to eq ['bob']
-        person_field = ActiveFedora.index_field_mapper.solr_name('foo__person', type: :string)
-        solr_result = ActiveFedora::SolrService.query("{!field f=id}#{@obj.id}", fl: "id #{person_field}").first
-        expect(solr_result).to eq("id" => @obj.id, person_field => ['bob'])
+        object2.update(person: ['dave'])
       end
-    end
 
-    describe "that already exists in the repo" do
-      before do
-        @release = MockAFBaseRelationship.create
-        @release.foo.person = "test foo content"
-        @release.save
-      end
-      describe "and has been changed" do
-        before do
-          @release.foo.person = 'frank'
-          @release.save!
-        end
-        it "saves the datastream." do
-          expect(MockAFBaseRelationship.find(@release.id).foo.person).to eq ['frank']
-          person_field = ActiveFedora.index_field_mapper.solr_name('foo__person', type: :string)
-          expect(ActiveFedora::SolrService.query("id:\"#{@release.id}\"", fl: "id #{person_field}").first).to eq("id" => @release.id, person_field => ['frank'])
-        end
-      end
-      describe "when trying to create it again" do
-        it "raises an error" do
-          expect { MockAFBaseRelationship.create(id: @release.id) }.to raise_error(ActiveFedora::IllegalOperation, "Attempting to recreate existing ldp_source: `#{@release.uri}'")
```

Comment:
Yes, it moved here: https://github.com/projecthydra/active_fedora/pull/1074/files#diff-2171cb3af303cbb38abe888472a92489R39